legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/protocols/pure/vX_X_X/models/metamodel_connection.pure

```diff
@@ -55,6 +55,18 @@ Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::a
 {
 }
 
+Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::authentication::GCPWorkloadIdentityFederationAuthenticationStrategy extends meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::authentication::AuthenticationStrategy
+{
+    workloadProjectNumber : String[1];
+    serviceAccountEmail : String[1];
+    gcpScope: String[1];
```

Comment:
Would we ever need to include more than one scope ?

Given that the metamodel is immutable, I'd rather that we allow multiple scopes in the metamodel.

Reply:
Say we need only a single scope for BigQuery. If all specifications have to specify the same scope value, should we simply hard code it in Engine ? (Think your first implementation did this).

So how about something like this - "Standard" scopes that are needed for BigQuery are automatically provided by Engine. If there is a need, users can specify one/more scopes through the model spec. In engine we can validate/uniqueify the scopes as needed.